src/sentry_log.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+
+#include "sentry_log.h"
+#include "sentry_options.h"
+
+#if defined(SENTRY_PLATFORM_ANDROID)
+
+#    include <android/log.h>
+void
+sentry_log_hook_default(sentry_level_t level, const char *message, va_list args)
+{
+    __android_log_vprint(ANDROID_LOG_INFO, "sentry-native", message, args);
+}
+
+#else
+
+void
+sentry_log_hook_default(sentry_level_t level, const char *message, va_list args)
+{
+    vfprintf(stderr, message, args);
+}
+
+#endif
+
+void
+sentry_log(sentry_level_t level, const char *message, ...)
+{
+    const sentry_options_t *_options = sentry_get_options();
```

Comment:
also here, we don’t have a convention to prefix with underscores, so:

```suggestion
    const sentry_options_t *options = sentry_get_options();
```

Ah oh, now I see that this comes from within the macro. Still a good idea to change it though.